scripts/rag_evaluation/create_evaluation_dataset/chunks_flatten_strategy/fhir_flattener.py

```diff
@@ -0,0 +1,140 @@
+import base64
```

Comment:
If this logic is taken from another open source code then you should add a comment here saying from where you got the code